src/components/Subheader360.js

```diff
@@ -1,15 +1,51 @@
 // @flow
 
 import React from 'react';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
 
 type Subheader360Props = {
+  /**
+   * The text that should be used for the subheader
+   */
   label: string;
+  /**
+   * If true, shows an information icon next to the subheader
+   */
+  description: boolean;
+  /**
+   * The text that should be shown in the tooltip when the information icon is hovered over
+   */
+  descriptionText?: string;
```

Comment:
Can we rename this to "tooltip" to match the property you added to the map facets?
Also, can we avoid having a separate description Boolean property and just show the icon (and the tooltip) if the descriptionText/tooltip string is set? You can have it default to null or undefined or '' instead of "No description text provided" which we would never want to accidentally show to the user. Does this make sense?

Reply:
Yeah, that makes sense. I just made these changes. Let me know if you want me to change anything else :)